pandas/_libs/tslibs/strptime.pyx

```diff
@@ -186,6 +186,7 @@ def array_strptime(
         bint iso_format = format_is_iso(fmt)
         NPY_DATETIMEUNIT out_bestunit
         int out_local = 0, out_tzoffset = 0
+        bint string_to_dts_succeeded = 0
```

Comment:
any particular reason this is changed from failed?  doesn't really matter to me, just curious

Reply:
it just simplifies the logic